controllers/conduit/proxy.go

```diff
@@ -65,6 +58,26 @@ func (i *Proxy) getEnvVars(allEnv []corev1.EnvVar) []corev1.EnvVar {
 			Name:  "NSM_NETWORK_SERVICE_NAME",
 			Value: common.LoadBalancerNsName(i.conduit),
 		},
+		{
+			Name:  "NSM_TRENCH_NAME",
+			Value: i.trench.ObjectMeta.GetName(),
+		},
+		{
+			Name:  "NSM_NSP_SERVICE_NAME",
+			Value: common.NspSvcName,
+		},
+		{
+			Name:  "NSM_NSP_SERVICE_PORT",
+			Value: strconv.Itoa(common.NspTargetPort),
+		},
+		{
+			Name:  "NSM_CONDUIT_NAME",
+			Value: i.conduit.ObjectMeta.GetName(),
+		},
+		{
+			Name:  "NSM_IP_FAMILY",
+			Value: common.GetIPFamily(i.trench),
```

Comment:
Why is it dual stack always? Shouldn't it be i.trench.Spec.IPFamily?

Reply:
Right, thanks, I just modified it in: https://github.com/Nordix/Meridio-Operator/blob/571e8b7c476f7eef2cfead0be5b3af4748eb8a67/controllers/common/models.go#L17